egui-d3d9/src/state.rs

```diff
@@ -91,7 +89,8 @@ impl Drop for DxState {
         // restore the previous state
         unsafe {
             expect!(
-                self.dev.SetTransform(D3DTS_WORLD, &self.original_world),
+                self.dev
+                    .SetTransform(D3DTRANSFORMSTATETYPE(0 + 256), &self.original_world),
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/direct3d9/d3dts-worldmatrix

For some reason the D3DTS_WORLD macro is gone on the new version?

https://learn.microsoft.com/en-us/windows/win32/direct3d9/d3dts-world